integrated_channels/sap_success_factors/models.py

```diff
@@ -207,24 +207,26 @@ def _payload_data(self, empty_value):
             completedTimestamp=self.completed_timestamp,
             grade=self.grade,
             # TODO: determine these values from the enrollment seat?
-            price=empty_value,
-            currency=empty_value,
-            creditHours=empty_value,
-            # These fields currently have no edX source, so remain empty.
-            totalHours=empty_value,
-            contactHours=empty_value,
-            cpeHours=empty_value,
-            instructorName=empty_value,
-            comments=empty_value,
+            # We might at a later date choose to send these values,
+            # but at this point we will omit them from the payload to avoid errors.
+            #
+            # price=empty_value,
```

Comment:
It would be nice if we defined a fieldset with all of these values, then we can iterate over, say, the `self` attributes and if any are found + contain values we could include them in the payload.